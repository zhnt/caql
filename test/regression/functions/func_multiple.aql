function first() {
    print("First function")
}

function second() {
    print("Second function")
}

first()
second()
first()
