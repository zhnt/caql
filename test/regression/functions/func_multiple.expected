First function
Second function
First function
